src/Components/CategorySelect.jsx

```diff
@@ -0,0 +1,79 @@
+import { fetchCategories } from "../api"
+import { useState, useEffect } from "react"
+import * as React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import MenuItem from '@mui/material/MenuItem';
+import Menu from '@mui/material/Menu';
+import { Link } from "react-router-dom";
+
+
+const CategorySelect = () => {
+
+    const [categoriesArray, setCategoriesArray] = useState([])
+    const [selectedIndex, setSelectedIndex] = useState(0);
+
+    useEffect(() => {fetchCategories().then((categories) => {
+        const categoryNames = categories.map((categoryObject) => {
+            return categoryObject.slug
+        })
+        setCategoriesArray(["view all", ...categoryNames])
```

Comment:
this could do with maybe a hover effect or something similar so it's clearer this is a dropdown menu